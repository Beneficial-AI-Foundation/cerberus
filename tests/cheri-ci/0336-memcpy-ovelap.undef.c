#include <stdlib.h>
#include <string.h>

int main()
{
    char *p=malloc(1);
    memcpy(p,p,1);
}
