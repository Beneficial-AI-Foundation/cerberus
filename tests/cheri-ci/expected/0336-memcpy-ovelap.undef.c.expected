Undefined {ub: "UB100", stderr: "", loc: "<other location: memcpy>"}
